src/tunacode/ui/screens/model_picker.py

```
@@ -245,19 +270,38 @@ def _rebuild_options(self) -> None:
+
+        recent_id_set: set[str] = set(recent_ids)
+
+        if recent_ids:
+            option_list.add_option(Option("── Recent ──", disabled=True))
+            for model_id in recent_ids:
+                option_list.add_option(self._make_model_option(model_id, get_model_pricing, format_pricing_display))
+            option_list.add_option(Option("── All Models ──", disabled=True))
+
         highlight_index = _choose_highlight_index(visible_models, self._current_model_id)
 
         for display_name, model_id in visible_models:
```

Ruff (0.14.14)

[warning] Loop control variable `display_name` not used within loop body

Rename unused `display_name` to `_display_name`

(B007)